jobs/build/advisories/Jenkinsfile

```diff
@@ -0,0 +1,259 @@
+                echo "out: ${out}"
+
+                rpm_advisory_id = buildlib.extractAdvisoryId(out)
+                echo "extracted rpm_advisory_id: ${rpm_advisory_id}"
+            }
+            stage("create image advisory") {
+                cmd = "-g openshift-${params.VERSION} "
+                cmd+= "create -k image "
+                cmd+= "--assigned-to ${params.ASSIGNED_TO} "
+                cmd+= "--manager ${params.MANAGER} "
+                cmd+= "--package-owner ${params.PACKAGE_OWNER} "
+                cmd+= "--impetus ${params.IMPETUS} "
+                cmd+= params.DATE ? "--date ${params.DATE} " : ""
+                cmd+= params.DRY_RUN ? "" : "--yes"
+
+                echo "elliott cmd: ${cmd}"
+                out = buildlib.elliott(cmd, [capture: true])
+                echo "out: ${out}"
+
+                image_advisory_id = buildlib.extractAdvisoryId(out)
+                echo "extracted image_advisory_id: ${image_advisory_id}"
+            }
+            stage("update job description with advisory links") {
+                currentBuild.description = params.DRY_RUN ? "[DRY_RUN]" : ""
+                currentBuild.description+= "https://errata.devel.redhat.com/advisory/${rpm_advisory_id}\n"
+                currentBuild.description+= "https://errata.devel.redhat.com/advisory/${image_advisory_id}"
+            }
+            stage("commit new advisories to ocp-build-data") {
+                cmd = "rm -rf ocp-build-data ; "
+                cmd+= "git clone --single-branch --branch openshift-${params.VERSION} git@gitlab.cee.redhat.com:openshift-art/ocp-build-data.git ; "
+                cmd+= "cd ocp-build-data ; "
+                cmd+= "sed -e 's/image:.*/image: ${image_advisory_id}/' -e 's/rpm:.*/rpm: ${rpm_advisory_id}/' -i group.yml ; "
+                cmd+= "git diff ; "
+                cmd+= "git add . ; "
+                cmd+= "git commit -m 'Update advisories on group.yml' ; "
+                cmd+= "git push origin openshift-${params.VERSION}"
+
+                echo "shell cmd: ${cmd}"
+                if (params.DRY_RUN) {
+                    out = "DRY RUN mode, command did not run"
+                } else {
+                    out = commonlib.shell(
+                        returnStdout: true,
+                        script: cmd
+                    )
+                }
+                echo "out: ${out}"
+            }
+            stage("login to bugzilla") {
+                withCredentials([usernamePassword(credentialsId: 'art-bugzilla', passwordVariable: 'bz_pass', usernameVariable: 'bz_user')]) {
+                    cmd = "bugzilla login ${env.usernameVariable} ${env.passwordVariable}"
+                    out = commonlib.shell(
+                        returnStdout: true,
+                        script: cmd
+                    )
+                    echo "out: ${out}"
+                }
+            }
+            stage("create placeholder bug") {
```

Comment:
why not use `elliott --group openshift-4.1 create-placeholder --kind rpm --use-default-advisory rpm` ?

Reply:
There was no special reason, just that I wrote that before https://gitlab.cee.redhat.com/openshift-art/tools/elliott/merge_requests/71 got merged

Reply:
Be careful with that. The bot won't have all of the group memberships that we have. So, things may fail silently.

Reply:
okay as long as it works, I think we can merge it, I can open another PR for using Elliott in the future.

Reply:
I just tested now and it failed :( I'll definitely change it to use the new `elliott create-placeholder` command !

Reply:
done! i'll have the opportunity to test it again later today (when creating release 4.1.13) ... if it passes, I'll remove the `[WIP]` from this PR ... otherwise, back to the drawing board :smile:

Reply:
it worked beautifully with `elliott create-placeholder`. Removing `[WIP]` !